README.md

```diff
@@ -120,18 +120,6 @@ You can use the following channels for getting help with Hazelcast:
   development team and other Hazelcast users.
 * [Stack Overflow](https://stackoverflow.com/tags/hazelcast)
 
-## Contributing
```

Comment:
Removed this section because it is outdated and covered separately in `CONTRIBUTING.md`